dnf/util.py

```diff
@@ -644,29 +644,51 @@ def _is_file_pattern_present(specs):
 
 
 def is_container():
+    """Returns true if DNF is running in a container,
+       false otherwise.
+    """
+    # This attempts to reimplement
+    # https://github.com/ostreedev/ostree-rs-ext/blob/b0d1c1e4748c8708149bb6c92998da623260c4c6/lib/src/container_utils.rs#L20
```

Comment:
This is OK by me, however I think we could also check first if `systemd-detect-virt` exists, if it does I think it's sane to just fork off `systemd-detect-virt -c`.

Reply:
And if we don't detect `systemd-detect-virt`, we use this logic?

Reply:
Also... this was used by is_bootc_container() so I think we can skip shipping this for now too. But I think once we need to tackle the kernel_install path (if we are doing it here) we might need to add these or similar.